### PR TITLE
Rob fixing some feature specs for #3300

### PR DESCRIPTION
**Pull request by rbarreca**
Branch to merge into main test_suite branch after reviews from @ivar and @thermistor

### CODE REVIEW

File: spec/support/sorting_helpers.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 def click_header(text)
-  find(:xpath, "//table[contains(@class, 'js-sticky')]/thead/tr/th[@data-column='#{text}']").click
+  find(:xpath, "//table[contains(@class, 'sortable')]/thead/tr/th[@data-column='#{text}']").click
```
Comment (rbarreca) [inline spec/support/sorting_helpers.rb]: Not super sure what's up here. There is a js-sticky table dynamically created that is prepended in DOM before the sortable table. The old tests were looking to it, but pointing the tests to the sortable table made them pass.

File: spec/features/admin/filter_products_spec.rb [hunk 2]
```diff
@@ -1,6 +1,11 @@
-require "spec_helper"
+require 'spec_helper'
+require 'support/multiselect_helpers'
 
-describe "Filter products", :js do
+RSpec.configure do |c|
+  c.include MultiselectHelpers
+end
```
Comment (rbarreca) [inline spec/features/admin/filter_products_spec.rb]: What is the right way to do helpers such as these?

Comment (thermistor) [reply spec/features/admin/filter_products_spec.rb]: See [spec_helper.rb](https://github.com/LocalOrbit/localorbit/blob/97a8d459925a571e9809522adef48196abc0edb7/spec/spec_helper.rb#L83)

Comment (rbarreca) [reply spec/features/admin/filter_products_spec.rb]: Have fixed this. Thanks.

File: spec/features/admin/managing_market_managers_spec.rb [hunk 3]
```diff
@@ -47,7 +47,10 @@
 
       open_email("new-user@example.com")
       expect(current_email.body).to have_content("You have been invited")
-      expect(current_email.body).to have_content(market.name)
+      expect(current_email.body).to have_content('Join Local Orbit')
+
+      # TODO: Revisit once bug #3308 fixed
+      # expect(current_email.body).to have_content(market.name)
```
Comment (ivar) [inline spec/features/admin/managing_market_managers_spec.rb]: unless there's an expectation to search the codebase for a string when starting work, I find it works best to just kill commented code and leave it for whoever starts on the future work to add their own specs as they find appropriate at that moment.

**Comment by rbarreca** [comment]
Cool, will remove. I like it.


<http://www.localorbit.com>
Rob Barreca

CEO | Local Orbit

office: (800) 598-3351 <8005983351>
mobile: (808) 224-1905 <8082241905>
site: www.localorbit.com
<https://www.facebook.com/localorbit> <https://www.twitter.com/localorbit>
<https://www.instagram.com/localorbit>

**Pull request merged**